app/views/part/test.scala.html

```diff
@@ -106,14 +108,17 @@
         <fieldset>
             <legend>@Messages("tryIt.meta")</legend>
             @for(metaProp <- metaProps) {
-                <div class="form-group">
-                    <label for="meta.@metaProp" class="col-sm-2 control-label">@metaProp</label>
-                    <div class="row">
-                        <div class="col-sm-5">
-                            <input name="meta.@metaProp" id="meta.@metaProp" type="text" class="form-control" />
+                @defining(part.registeredParamsView.find(_.inputNameJs == s"meta.$metaProp")) { metaParam =>
+                    <div class="form-group">
+                        <label for="meta.@metaProp" class="col-sm-2 control-label">@metaProp</label>
+                        <div class="row">
+                            <div class="col-sm-5">
+                                <input name="meta.@metaProp" id="meta.@metaProp" type="text" class="form-control @{metaParam.fold("optional")(_.requiredCls)}"/>
```

Comment:
which view? The fold(optional) means is there is no such registered parameter, make the meta param optional